course6/string_ariel/str.h

```diff
@@ -0,0 +1,193 @@
+#pragma once
+
+#include <iostream>
+#include <cstring>
+
+class Str
```

Comment:
Wrap it into a namespace. Get into a habit of using namespaces. All the production code on the planet is inside of namespace, so it's good to get some experience with it as well.

Reply:
Ok, will do on my next PR